common/reviews/api/rush-lib.api.md

```diff
@@ -137,6 +143,35 @@ export class ExperimentsConfiguration {
     get configuration(): Readonly<IExperimentsJson>;
 }
 
+// @beta (undocumented)
+export type IBuildCacheJson = ICloudBuildCacheJson | ILocalBuildCacheJson;
+
+// Warning: (ae-forgotten-export) The symbol "IBaseBuildCacheJson" needs to be exported by the entry point index.d.ts
```

Comment:
@iclanton FYI I'd like to tune up `rush-lib.api.md` a bit before we release this feature, to eliminate some of these warnings and better distinguish experimental APIs from production APIs.  However PR #2900 has accumulated a lot of work already, so I've decided to merge it and use a clean PR for the tune up.